include/network/Network.h

```diff
@@ -150,21 +166,46 @@ template <typename T> class Network : public NetworkBase<T> {
             throw std::invalid_argument("Padded weights width cannot be less than the unpadded.");
 
         /// we need to copy everything here since we do not want to have an implicit copy of 'this'
+
         const int padded_input_width = get_input_width();
         const int network_width = get_network_width();
         const int padded_output_width = get_output_width();
+        const int input_width_divisibly_by = 16;
+
+        // to replicate the behaviour of bias (we only do W*x, not W*x+bias), the input has (unpadded_input_width +
+        // one_padded_input_width + zero_padded_input_width) elements, where
```

Comment:
really unpadded_input_width + one_padded_input_width ? Isn't one_padded_input_width already the sum of unpadded and 1's?

Reply:
change, the comment is wrong